SoftLayer/CLI/subnet/route.py

```diff
@@ -17,10 +18,16 @@
 @click.argument('identifier')
 @click.option('--target', type=click.Choice(['vlan', 'ip', 'hardware', 'vsi']),
               help='choose the type. vlan, ip, hardware, vsi')
-@click.option('--target-id', help='The identifier for the destination resource to route this subnet to. ')
+@click.option('--target-resource', help='An appropriate identifier for the specified $type. '
+                                        'Some types have multiple identifier. ')
 @environment.pass_env
-def cli(env, identifier, target, target_id):
+def cli(env, identifier, target, target_resource):
     """Assigns the subnet to a target."""
```

Comment:
Update this documentation to include some usage examples similar to https://github.ibm.com/softlayer/softlayer-cli/blob/master/plugin/commands/subnet/route.go  should also include a link to https://sldn.softlayer.com/reference/services/SoftLayer_Network_Subnet/route/ since this command takes a variety of options